docs/src/main/sphinx/sql/delete.rst

```diff
@@ -34,5 +34,9 @@ Delete all orders::
 Limitations
 -----------
 
+If a ``WHERE`` clause is
```

Comment:
This is wrong @jhlodin  .. instead this is a limitation that applies for each connector. So I suggest to add it as subsection about delete for each of these connectors as well as postgresql and sqlserver. Ping me to chat if you need clarifcation or want to discuss

Reply:
Created fragment about this SQL `DELETE` limitation and added to JDBC connector docs.